src/pages/MonthlyGood/HowItWorks.vue

```diff
@@ -68,16 +44,10 @@ export default {
 
 </script>
 
-<style lang="scss" scoped>
-@import 'settings';
+<style scoped>
 
-.how-works-section-wrapper {
-	.mg-image {
-		fill: $white;
-		width: 3.125rem;
-		display: block;
-		margin: 0 auto 1rem auto;
-	}
+#rteContent >>> figure{
+	margin-bottom: 0;
```

Comment:
@roger-in-kiva and I have done some diving into Tailwinds and it's not-prose declaration and have some ideas where the extra margin this fixes is coming from. However, the alterations to fix the margins on the bottom of imges embedded in rich text fields will be a much broader change so I've suggested this local hack for now. ref. https://kiva.atlassian.net/browse/VUE-1380 as this is a known issue.